AssemblerLib/LocalDataInitializer.h

```diff
@@ -240,18 +242,19 @@ class LocalDataInitializer
 
     /// Sets the provided data_ptr to the newly created local assembler data.
     void operator()(
-            const MeshLib::Element& e,
+            std::size_t const mesh_item_id,
```

Comment:
New comment is
```
/// Sets the provided \c data_ptr to the newly created local assembler data.
///
/// \attention
/// The index \c id is not necessarily the mesh item's id. Especially when
/// having multiple meshes it will differ from the latter.
```